jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -587,6 +589,12 @@ function configure_jbpm() {
     fi
 }
 
+function configure_optaplanner() {
+    if [ "${JBOSS_PRODUCT}" = "rhpam-kieserver" ]; then
```

Comment:
the optaplanner extension is available on rhdm as well.

Reply:
Done!